src/ig/IGExporter.ts

```diff
@@ -75,50 +82,27 @@ export class IGExporter {
   }
 
   /**
-   * Initializes the ImplementationGuide JSON w/ data from the package.json
+   * Initializes the ImplementationGuide JSON w/ data from the configuration YAML
    *
    * @see {@link https://confluence.hl7.org/pages/viewpage.action?pageId=35718629#NPMPackageSpecification-PackageManifestpropertiesforIGs}
    */
-  private initIG(): void {
-    const packageJSON = this.pkg.packageJSON;
-    // TODO: Consider adding a generated file warning when the IG Publisher supports JSON5-style comments
+  private initIG() {
+    // first, properties that can be directly used without much trouble
     this.ig = {
       resourceType: 'ImplementationGuide',
-      id: packageJSON.name,
-      url: `${packageJSON.canonical}/ImplementationGuide/${packageJSON.name}`,
-      version: packageJSON.version,
+      id: this.config.id,
+      url: `${this.config.url}/ImplementationGuide/${this.config.name}`,
```

Comment:
I believe this can now just be `this.config.url` because the `url` will be set to the default Nick described on the import side.